fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -103,18 +103,35 @@ def self.print_table(actions)
     # Lane chooser if user didn't provide a lane
     # @param platform: is probably nil, but user might have called `fastlane android`, and only wants to list those actions
     def self.choose_lane(ff, platform)
+      available = ff.runner.lanes[platform].to_a
+      if available.empty?
+        UI.error "It looks like you don't have any lanes to run just yet. Check out how to get started here: https://github.com/fastlane/fastlane 🚀"
+        exit
+      end
       loop do
-        UI.error "You must provide a lane to drive. Available lanes:"
-        available = ff.runner.available_lanes(platform)
-
+        UI.message "Welcome to fastlane! Here's what your app is setup to do:"
+        rows = []
         available.each_with_index do |lane, index|
-          UI.message "#{index + 1}) #{lane}"
+          rows << [index + 1, lane.last.pretty_name, lane.last.description.first]
```

Comment:
now that we've moved the collection of available lanes outside of the loop, we could move this one outside as well.